### PR TITLE
Addressing the TODOs

### PR DESCRIPTION
**Pull request by netmilk**
- Monetization Python and CLI examples
- Local Development
- Actorization Example (curl)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1327,9 +1327,9 @@ by the subsequent Actors are taken from the calling Actor's allowance.
 This enables Actor economy, where Actors hierarchically pay other Actors or external APIs
 to perform parts of the job.
 
-#### Node.js
+As an Actor developer, you can charge the current user of the Actor a specific amount of USD
```
Comment (jancurn) [inline README.md:1330]: nit: missing dot

File: README.md [hunk 2]
```diff
@@ -1625,9 +1659,53 @@ The SDK is currently available for Node.js, Python, and CLI.
 
 ### Local development
 
-TODO (Adam): Explain basic workflow with `apify` - create, run, push etc. Move the full local support for Actors
- to ideas (see https://github.com/apify/actor-specs/pull/7/files#r794681016 )
+Actor Programming Model is language agnostic, but the framework has native support for detection of Javascript and Python languages.
```
Comment (jancurn) [inline README.md:1662]: I think Programming Model should be lower-case, it's upper case only in the title

File: README.md [hunk 3]
```diff
@@ -1625,9 +1659,53 @@ The SDK is currently available for Node.js, Python, and CLI.
 
 ### Local development
 
-TODO (Adam): Explain basic workflow with `apify` - create, run, push etc. Move the full local support for Actors
- to ideas (see https://github.com/apify/actor-specs/pull/7/files#r794681016 )
+Actor Programming Model is language agnostic, but the framework has native support for detection of Javascript and Python languages. 
+
+Tip: [Apify CLI](https://docs.apify.com/cli/docs/next/reference#apify-create-actorname) comes with [convenient tamplates](https://apify.com/templates) you can bootsrap an Actor with for Python and Javascript.
```
Comment (jancurn) [inline README.md:1664]: some typos here

**Review by jancurn — APPROVED** [review]
Looks good, just tiny comments, thanks

**Review by janbuchar — APPROVED** [review]
LGTM

**Comment by netmilk** [comment]
Perfect. I addressed all the comments. If you think it's OK to merge, let's do it, please.

**Pull request merged**